### ISSUE TITLE
Consistent Release Archives

### ISSUE THREAD

**Issue by elee** [spiffe/spire#1309]
The smallest of nit picks but potentially helpful feedback: consistently putting the `spire-{agent,server}` binaries in either the root of the `.tgz` or in `bin` would be helpful to downstream consumers who package up these binary releases for testing.  Between 0.8.4 and 0.9.0 the binaries have moved between the root of the archive and `bin`

* **Version**: 0.8.4 / 0.9.0
* **Platform**:  n/a
* **Subsystem**: server, agent

**Comment by evan2645** [comment]
Thanks for opening this @elee 

Yes, sorry about this... The tarball contents are historically stable, though they were littered with some superfluous stuff and we decided to shake up the organization a little bit so that it makes more sense. I don't expect this to happen very often (ever?), and this is the first time we've done it

That said, we chose 0.9.0 to do this we felt a ".0" was the right time to introduce such a change (given that it's minorly disruptive).

Hopefully you prefer the new organization over the old one :)

Going to go ahead and close this out. There definitely won't be any more changes on this front for quite some time. Thanks for the poke though @elee!
